kvbc/include/kvbc_adapter/replica_adapter.hpp

```diff
@@ -109,7 +109,19 @@ class ReplicaBlockchain : public IBlocksDeleter,
   std::optional<categorization::Updates> getBlockUpdates(BlockId block_id) const override final {
     return reader_->getBlockUpdates(block_id);
   }
-
+  // find the first block which has the given sequence number in its metadata
```

Comment:
Can multiple blocks have the same sequence number in their metadata?

Reply:
yes, but they will all be placed sequentially one after another, so as long as we stop on the first one it's ok